ts/src/bybit.ts

```diff
@@ -2765,11 +2775,25 @@ export default class bybit extends Exchange {
         }
         const marketId = this.safeString (trade, 'symbol');
         market = this.safeMarket (marketId, market, undefined, 'spot');
+        const symbol = market['symbol'];
         let fee = undefined;
         const feeCost = this.safeString (trade, 'execFee');
         if (feeCost !== undefined) {
-            const feeToken = this.safeString (trade, 'feeTokenId');
-            const feeCurrency = this.safeCurrencyCode (feeToken);
+            let feeCurrency = undefined;
+            const [ base, quote ] = symbol.split ('/');
+            if (Precise.stringGt (feeCost, '0')) {
+                if (side === 'buy') {
+                    feeCurrency = base;
+                } else {
+                    feeCurrency = quote;
+                }
+            } else {
+                if (side === 'buy') {
```

Comment:
also not sure about this  <0 >0 logic, I just found this int he docs but that does not translate to this logic
![image](https://github.com/ccxt/ccxt/assets/43336371/cfe39afa-ed63-488c-a984-d1b8759b095f)